Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -447,6 +447,10 @@ - (void)configureViews
                          }];
 
     [self updateBubbleColor];
+
+    // If we're stroking the bubble edge, ensure the stroke
```

Comment:
TIOLI, but rather than adding the bubbleStrokeView and then juggling it's position, can instead just add the bubbleStrokeView later? Just trying to keep our concerns colocated.

A quick grep shows we only use the bubbleStrokeView for contactShares - is that right?

Reply:
> can instead just add the bubbleStrokeView later?

Ordering matters as we insert subviews into the stack view, so we can't move contact share handling here.  If we move adding the stroke view here, I think we we'd duplicate logic that I'd rather DRY.  

It's admittedly not ideal.  If you have a proposal, I'd be happy to see it.